Firmware/Marlin_main.cpp

```diff
@@ -5623,7 +5623,7 @@ if(eSoundMode!=e_SOUND_MODE_SILENT)
                 SERIAL_PROTOCOLPGM("\nZ search height: ");
                 SERIAL_PROTOCOL(MESH_HOME_Z_SEARCH);
                 SERIAL_PROTOCOLLNPGM("\nMeasured points:");
-                for (uint8_t y = MESH_NUM_Y_POINTS-1; y >= 0; y--) {
+                for (int16_t y = MESH_NUM_Y_POINTS-1; y >= 0; y--) {
```

Comment:
this looks like something else has not been optimized - it doesn't make sense to keep the int16_t at this spot if `MESH_NUM_Y_POINTS` never exceeds 255.